docker-compose.yml

```diff
@@ -101,4 +101,4 @@ volumes:
             o: bind
             # This directory specified where InvenTree source code is stored "outside" the docker containers
             # By default, this directory is one level above the "docker" directory
-            device: ${INVENTREE_EXT_VOLUME:-../}
+            device: ${INVENTREE_EXT_VOLUME:-./}
```

Comment:
Question: Any specific reason for the move?

Reply:
Yep. Previously the docker build was performed from within the `./docker` directory, and involved re-downloading the source code from git based on the specified tag. However as we are now building from the github workflow, we have already defined the context of the built image (latest / stable / tag) and so we can use the simpler process of copying source files into the image.

However this needs to be performed from the top-level directory as you cannot `COPY` files from a directory above

Reply:
Seems more efficient, I like it!